### PR TITLE
Buttons on the page now redirect to other pages.

### PR DESCRIPTION
**Pull request by Phoebe-Khokgawe**

### CODE REVIEW

**Review by roonil-wazlib — APPROVED** [review]

File: portfolio/frontend/src/containers/CreateCompetition/CreateCompetition.module.css [hunk 1]
```diff
@@ -58,23 +54,49 @@
...
-	justify-content: space-between;
+	justify-content: space-evenly;
 	margin-top: 10%;
 	margin-left: 27%;
 }
 
-.button {
-	width: 60%;
-	height: 40%;
+.BlueButton {
```
Comment (leonnemets) [inline portfolio/frontend/src/containers/CreateCompetition/CreateCompetition.module.css:71]: Can the common attributes for Blue and White button be factored out

Comment (Phoebe-Khokgawe) [reply portfolio/frontend/src/containers/CreateCompetition/CreateCompetition.module.css:71]: I will look into that

Comment (Phoebe-Khokgawe) [reply portfolio/frontend/src/containers/CreateCompetition/CreateCompetition.module.css:71]: fix this in 0bf80d684ad4b8dad4d9ba9bbb2ecd87a0e53494

Comment (leonnemets) [reply portfolio/frontend/src/containers/CreateCompetition/CreateCompetition.module.css]: very nice

**Review by leonnemets — APPROVED** [review]

**Pull request merged**